hibp.js

```diff
@@ -17,25 +17,23 @@ const HIBP = {
 
     const sha1 = getSha1(email);
     const sha1Prefix = sha1.slice(0, 6);
-    const url = `${AppConstants.HIBP_STUB_API_ROOT}/breachedaccount/range/${sha1Prefix}`;
+    const url = `${AppConstants.HIBP_STAGE_API_ROOT}/range/${sha1Prefix}?code=${AppConstants.HIBP_STAGE_API_TOKEN}`;
```

Comment:
I'm guessing the `HIBP_STAGE_API_TOKEN` is just a basic alpha-numeric string and doesn't need to be special URL-encoded?